### DISCUSSION TITLE
Make one post per development update

### DISCUSSION THREAD

**Discussion by StableInfo** [vladmandic/sdnext#2217] [Ideas]
I just saw the development update post,
Its impressive.
But it's locked, no discussion allowed, arguing, more ideas, feedback.
My suggestion is to make a new post every update and number it (Dev update N°46..)
I want to see people reactions to changes: people thankful, people adding on things,more suggestions. That kind of post should not be locked.
Or maybe keep both posts, the specific ones (open) and keep the general locked one.

Furthermore, the changes should be displayed in the UI, in some tab, (maybe Imissed it if its already there?) and that tab has a link to the specific post where users can discuss further.

**Comment by brknsoul** [comment]
Why not make your own, linking the relevant dev post?

**Comment by vladmandic** [comment]
adding changelog to ui is a good idea - done!
ui -> system -> changelog

regarding separate post vs individual posts - like @brknsoul said, anyone can create a new thread, i don't think me creating it would make a big difference.
